packages/amplify_analytics_pinpoint/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.0.1-dev.3 (2020-10-08)
```

Comment:
This is really driving the point hope that it's a pre-release 😀.

Anything below 1.0.0 is acceptable. I'd drop the -dev business and cut a minor.

0.1.0?

For reference, the initial iOS and Android releases were 0.9, signifying that they were working towards 1.0.0, but were not quite there yet. We ended up having 0.10.x etc., before we pinned stable APIs in 1.0.0.

Reply:
Agree. We can change it to 0.x.x when we announce our release candidate with DataStore and API along with other improvements.

Reply:
I don't think there is anything wrong with driving a point home when it comes to OSS... but if we want to drop the -dev suffix, that's fine by me.

Reply:
There is also a pub.dev standards which consider non pre-id based release numbers as "releases" and with pre-id or tagged releases as pre-releases. e.g. https://pub.dev/packages/artemis/versions

Reply:
I think from @jamesonwilliams point, this kind of versioning probably lowers developer confidence in adoption. So maybe when we are closer to almost stable, switching to 0.x.x will indicate that better. Something to consider though I dont have a strong opinion either way.